example/serializers.py

```diff
@@ -1,6 +1,12 @@
 from datetime import datetime
+from packaging import version
```

Comment:
Yep, for the record, `packaging` is now a requirement of the recent `setuptools` (coming since python 3.5).

I've considered `distutils.version` which is built in but its classes are not documented (cf. https://docs.python.org/2/distutils/apiref.html#module-distutils.version) and seems less powerful than `packaging.version`

After all that code could be dropped as soon as we drop support of DRF 3.1 and 3.2.